src/server/migrations/migrateDatabase.js

```diff
@@ -0,0 +1,191 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+const db = require('../models/database').db;
+const Migration = require('../models/Migration');
+const { compareSemanticVersion } = require('../util');
+const { log } = require('../log');
+
+// file needed to run database transaction
+const requiredFile = [];
+
+/**
+ * @param migrationItems extracted from registerMigration.js
+ * @returns {Array} of migration pair
+ */
+function printMigrationList(migrationItems) {
+	const list = [];
+	for (const item of migrationItems) {
+		list.push(`${item.fromVersion} -> ${item.toVersion}`);
+	}
+	return list;
+}
+
+/**
+ * Create an array of unique versions;
+ * @param migrationItems extracted from registerMigration.js
+ * @returns {*[]} array of unique versions
+ */
+function getUniqueKeyOfMigrationList(migrationItems) {
```

Comment:
"unique key" isn't a great name for this either, since you aren't treating it like a key. Maybe something closer to `uniqueVersions`? But I'm not sold on that name either.